invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/oaipmh/details/LinksTable.js

```diff
@@ -79,107 +107,111 @@ function LinksTable({ data }) {
+                    <a
+                      href={listIdentifiers}
+                      target="_blank"
+                      title={i18next.t("Opens in new tab")}
+                    >
+                      {listIdentifiers}
+                    </a>
+                  </Table.Cell>
+                  <Table.Cell width={3} textAlign="right">
+                    <CopyButton text={listIdentifiers} />
+                  </Table.Cell>
+                </Table.Row>
+              </Table.Body>
+            </Table>
+            {!isEmpty(error) && (
+              <ErrorMessage
+                {...error}
+                removeNotification={this.resetErrorState}
+              />
+            )}
+          </>
+        }
+      </>
+    );
   }
-
-  return (
-    <>
-      {
-        <>
-          <Grid>
-            <Grid.Row />
-            <Grid.Row>
-              <Grid.Column width={3}>
-                <Header as="h2">{i18next.t("Links")}</Header>
-              </Grid.Column>
-              <Grid.Column width={13} textAlign="right">
-                <span className="mr-10">{i18next.t("Format")}</span>
-                <Dropdown
-                  options={formats}
-                  floating
-                  selection
-                  defaultValue={defaultPrefix}
-                  onChange={prefixOnChange}
-                />
-              </Grid.Column>
-            </Grid.Row>
-          </Grid>
-          <Table unstackable>
-            <Table.Body>
-              <Table.Row key="list-records">
-                <Table.Cell width={3}>
-                  <b>{i18next.t("List records")}</b>
-                </Table.Cell>
-                <Table.Cell width={10} textAlign="left">
-                  <a
-                    href={listRecords}
-                    target="_blank"
-                    title={i18next.t("Opens in new tab")}
-                  >
-                    {listRecords}
-                  </a>
-                </Table.Cell>
-                <Table.Cell width={3} textAlign="right">
-                  <CopyButton text={listRecords} />
-                </Table.Cell>
-              </Table.Row>
-              <Table.Row key="list-identifiers">
-                <Table.Cell width={3}>
-                  <b>{i18next.t("List identifiers")}</b>
-                </Table.Cell>
-                <Table.Cell width={10} textAlign="left">
-                  <a
-                    href={listIdentifiers}
-                    target="_blank"
-                    title={i18next.t("Opens in new tab")}
-                  >
-                    {listIdentifiers}
-                  </a>
-                </Table.Cell>
-                <Table.Cell width={3} textAlign="right">
-                  <CopyButton text={listIdentifiers} />
-                </Table.Cell>
-              </Table.Row>
-            </Table.Body>
-          </Table>
-        </>
-      }
-    </>
-  );
 }
 
-LinksTable.propTypes = {
```

Comment:
why are props removed? eslint should complain about it, we should add eslint-config-invenio if it is not already there